README.md

```diff
@@ -1 +1,57 @@
-# cohort8_structure
\ No newline at end of file
+# cohort 8 ;)
+## Overview
+In this cohort, we utilize learning resources from the [practical data science course](http://www.datasciencecourse.org/lectures/), with the goal of solving a data science problem, starting from data collection to model inference. 
+
+The cohort would last 16 weeks with 15 weeks of lectures and lab work and a break on week 9. We would select relevant lectures from the [practical data science course](http://www.datasciencecourse.org/lectures/), study them carefully and use the learned concepts in the labs.
+
+There would be a series of labs, with each building up to a final solution to a chosen problem. For the labs, we have chosen to work on a fancy problem that involves reddit posts; 
+
+ ```
+ Given a reddit post, can you predict the subreddit to which it belongs? 
+ ```
+
+Working on these labs would expose us to the art of building a data science project from the ground up, starting from data collection and completing the process with a model for inference.
+
+Additionally, participants are encouraged to work on a final project, exercising skills acquired from tinkering with the labs.
+
+
+## Selected Lectures
+Week |Date |  Topic | Resources | Instructors
+|---|---| ---| --- | --|
+1| 20-May |  Introduction to Data Science | [Slide](http://www.datasciencecourse.org/slides/intro.pdf), [Note](http://www.datasciencecourse.org/notes/intro/) | -
```

Comment:
@ldfrancis some of the slides links are broken. Please have a look.

Reply:
Oh